agent/roles/manifests/templates/appliance-config_yaml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1beta1
+kind: ApplianceConfig
+ocpRelease:
+  version: {{ version }}
+  channel: candidate
+  cpuArchitecture: x86_64
```

Comment:
Does the appliance support arm64? A recent PR brought in support for arm64: https://github.com/openshift-metal3/dev-scripts/pull/1518